locales/base/translation.json

```diff
@@ -1772,6 +1772,24 @@
     "selectTokenLabel": "Select token",
     "tokenBottomSheetTitle": "Select Token",
     "onNetwork": "on {{networkName}}",
+    "crossChainNotification": "Swapping tokens between different networks",
+    "crossChainFeeWarning": {
+      "title": "You need more {{tokenSymbol}}",
+      "body": "You need more {{tokenSymbol}} on {{networkName}} to pay for cross-chain swap fees. Add {{tokenAmount}} more {{tokenSymbol}} to continue."
+    },
+    "insufficientBalanceWarning": {
+      "title": "You need more {{tokenSymbol}}",
+      "body": "Insufficient {{tokenSymbol}} balance. Try a smaller amount or choose a different asset."
+    },
+    "unsupportedTokensWarning": {
+      "title": "Incompatible Selection",
+      "body": "The tokens selected cannot be swapped between. Please choose another token that is compatible with your selection."
```

Comment:
the second phrase sounds kind of weird to me. choose another token that is compatible with my selection?
maybe we can simplify it to just "Please choose another token pair"?